packages/frontend-2/pages/server-management/index.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink
+        :to="'/server-management'"
+        name="Server Management"
+      ></HeaderNavLink>
+    </Portal>
+    <div
+      class="flex flex-col md:flex-row space-y-2 space-x-2 justify-between mb-4 md:items-center"
+    >
+      <div>
+        <h5 class="h4 font-bold">Your server at a glance</h5>
+      </div>
+    </div>
+
+    <div class="grid grid-cols-1 md:grid-cols-2 gap-6 mt-10">
+      <Card :server-info="serverData" @cta-clicked="showDialog = true" />
+
+      <SettingsDialog
+        ref="settingsDialog"
+        v-model:open="showDialog"
+        title="Edit Settings"
+        :buttons="[
+          {
+            text: 'Cancel',
+            props: { color: 'secondary', fullWidth: true, outline: true },
+            onClick: closeDialog
+          },
+          {
+            text: 'Save',
+            props: { color: 'primary', fullWidth: true, outline: false },
+            onClick: saveSettings
+          }
+        ]"
+        @server-info-updated="fetchData"
+      />
+
+      <Card :server-info="userData" />
+      <Card :server-info="projectData" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { graphql } from '~~/lib/common/generated/gql'
+import { useQuery } from '@vue/apollo-composable'
+import { ref, computed, onMounted, reactive } from 'vue'
+import Card, { CardInfo } from '../../components/server-management/Card.vue'
+import SettingsDialog from '../../components/server-management/SettingsDialog.vue'
+import {
+  ServerIcon,
+  UsersIcon,
+  EnvelopeIcon,
+  ChartBarIcon,
+  HomeIcon
+} from '@heroicons/vue/24/solid'
+import { useRouter } from 'vue-router'
+
+interface GithubRelease {
+  url: string
+  assets_url: string
+  upload_url: string
+  html_url: string
+  id: number
+  node_id: string
+  tag_name: string
+}
+
+interface SettingsDialogRef {
+  onSubmit: () => void
+}
+
+const router = useRouter()
+const showDialog = ref(false)
+const settingsDialog = ref<SettingsDialogRef | null>(null)
+const latestVersion = ref<string | null>(null)
+const isLatestVersion = ref<boolean>(false)
```

Comment:
redundant type argument, it's already clear it's a boolean from the `false` value